src/utils/handleTime.js

```diff
@@ -4,6 +4,10 @@ export default function handleTime(eventStart, eventEnd = null) {
   let start = moment(eventStart);
   let end;
 
+  if (eventStart === null && eventEnd === null) {
+    return null;
+  }
+
```

Comment:
You could also write ```!eventStart && !eventEnd``` to cover the case if eventStart is `undefined` instead of `null`.

Reply:
`addNewEvent` case in `eventsReducer.js`, creates a new event and uses `null` for start and end dates. Created this logic for that specific use case but you're right. 

Updated with commit cc650fd

Reply:
Ah ok, I wasn't sure if it will always be `null`. But nice to see you changed it:)